spec/services/proofing/lexis_nexis/phone_finder/proofing_spec.rb

```diff
@@ -28,9 +28,19 @@
 
   describe '#proof' do
     context 'when the response is a success' do
-      it 'is a successful result' do
+      it 'is a successful rdp1 result' do
         stub_request(:post, verification_request.url).
-          to_return(body: LexisNexisFixtures.phone_finder_success_response_json, status: 200)
+          to_return(body: LexisNexisFixtures.phone_finder_rdp1_success_response_json, status: 200)
+
+        result = instance.proof(applicant)
+
+        expect(result.success?).to eq(true)
+        expect(result.errors).to eq({})
+      end
+
+      it 'is a successful rdp2 result' do
+        stub_request(:post, verification_request.url).
+          to_return(body: LexisNexisFixtures.phone_finder_rdp2_success_response_json, status: 200)
```

Comment:
Do you think it is worthwhile to create separate RDP1 and RDP2 fixture, and thus separate tests?
* The tests are otherwise identical in the case of `passed` responses
* We don't yet have an official `failed` response from Monica yet (I recently asked for one)
* A `failed` fixture will allow us to test if PII has been removed

Reply:
I think that this makes sense. I think a context for RDP1 and RDP2 makes sense at least until we are confident we are fully migrated to RDP2

Reply:
Thank you for the clarity. Changed!

Reply:
I do not seem to be getting an example `fail` case from LN so I guess I am ready to merge the tests I have.